src/components/BookList.jsx

```diff
@@ -1,60 +1,35 @@
-import { useState } from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { useSelector } from 'react-redux';
 import Card from './Card';
 import AddBook from './AddBook';
 
 function BookList() {
-  const [bookList, setBook] = useState([
-    {
-      id: 1,
-      category: 'Action',
-      name: 'The Hunger Games',
-      director: 'Suzanne Collins',
-      completed: '64%',
-      chapter: 'Chapter 17',
-    },
-    {
-      id: 2,
-      category: 'Science Fiction',
-      name: 'Dune',
-      director: 'Frank Herbert',
-      completed: '8%',
-      chapter: 'Chapter 3: A Lession',
-    },
-    {
-      id: 3,
-      category: 'Economy',
-      name: 'Capital in the Century',
-      director: 'Suzanne Collins',
-      completed: '0%',
-      chapter: 'Introduction',
-    },
-  ]);
-
-  const addBook = (newBook) => {
-    setBook([newBook, ...bookList]);
-  };
-
-  const handleDelete = (nid) => {
-    setBook(bookList.filter((todo) => todo.id !== nid));
-  };
+  const { bookItems, count } = useSelector((book) => book.bookItems);
+  if (count < 1) {
+    return (
+      <>
+        <div className="h-screen">
+          <p className="text-center">Empty</p>
+        </div>
+        <AddBook />
+      </>
+    );
+  }
 
   return (
-    <div>
-      {bookList.map((todo) => (
+    <>
+      {bookItems.map((item) => (
         <Card
-          key={todo.id}
-          category={todo.category}
-          name={todo.name}
-          director={todo.director}
-          completed={todo.completed}
-          chapter={todo.chapter}
-          deleteBook={() => handleDelete(todo.id)}
+          key={item.id}
+          id={item.id}
+          title={item.title}
+          author={item.author}
+          category={item.category}
         />
       ))}
```

Comment:
- [ ] I also cannot remove any book from the page, whether it is the initial book or the one added. The remove functionality does not at all. Kindly fix that.